recipe/meta.yaml

```diff
@@ -1,26 +1,28 @@
-{% set version = "1.4.0" %}
+{% set version = "1.5.2" %}
 
 package:
   name: pkgconfig
   version: {{ version }}
 
 source:
   url: https://github.com/matze/pkgconfig/archive/v{{ version }}.tar.gz
-  sha256: 38c5de8392f4acfe7f36b25d113c496d68f534e983afdbd0ba7240c8c475b161
+  sha256: 4df0845fa1936f232b9afd951943f452acab0239de6eda753c914c199de5aab3
 
 build:
-  number: 3
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - pkg-config  # [unix]
+    - poetry
   host:
     - python
     - pip
     - nose >=1.0
+    - poetry
```

Comment:
this for sure needs to be here, but `nose` probably can go (and is becoming problematic on modern python)

Reply:
Also, the `pyproject.toml` only claims `poetry_core`:

https://github.com/matze/pkgconfig/blob/v1.5.2/pyproject.toml#L2

It looks like this is a less-complicated version than the whole env/build/publish CLI tool:

https://github.com/conda-forge/poetry-core-feedstock/blob/master/recipe/meta.yaml#L24

Reply:
my bad, `poetry-core` in conda-forge :blush:

Reply:
Thanks! That was the fix!